From d38cc5eb0137848b0e393a1186910dc9ddaa26e1 Mon Sep 17 00:00:00 2001
From: Rosen Penev <rosenp@gmail.com>
Date: Thu, 16 Mar 2023 19:28:52 +0530
Subject: [PATCH 103/281] OpenWrt: 253-ksmbd-config.patch

Kconfig: add tristate for OID and ASNI string

target/linux: add help text to kernel config

These options will be used for ksmbd. Once kernel 5.15
makes it in, this patch can go away.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

Change-Id: Ib52a8ad7888ef8080eac53853c24d3abaccca21f
Signed-off-by: Ram Chandra Jangir <quic_rjangir@quicinc.com>
---
 init/Kconfig | 2 +-
 lib/Kconfig  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/init/Kconfig b/init/Kconfig
index 4fc578f97cd8..facf8b2d5d40 100644
--- a/init/Kconfig
+++ b/init/Kconfig
@@ -2007,7 +2007,7 @@ config PADATA
 	bool
 
 config ASN1
-	tristate
+	tristate "ASN1"
 	help
 	  Build a simple ASN.1 grammar compiler that produces a bytecode output
 	  that can be interpreted by the ASN.1 stream decoder and used to
diff --git a/lib/Kconfig b/lib/Kconfig
index 3dc4cb534b30..9e2232f52e72 100644
--- a/lib/Kconfig
+++ b/lib/Kconfig
@@ -637,7 +637,7 @@ config LIBFDT
 	bool
 
 config OID_REGISTRY
-	tristate
+	tristate "OID"
 	help
 	  Enable fast lookup object identifier registry.
 
-- 
2.17.1

